executor/workflows/tasks.py

```diff
@@ -254,16 +259,66 @@ def test_workflow_notification(account_id, workflow, message_type):
             for step, all_task_results in all_step_executions.items():
                 for result in all_task_results:
                     playbook_execution_log = PlaybookExecutionLog(
-                        playbook=playbook,
-                        playbook_step=step,
-                        playbook_task_definition=result['task'].proto,
-                        playbook_task_result=result['task_result_proto'],
+                        playbook=p_proto,
+                        step=step.proto,
+                        task=result['task'].proto,
+                        task_execution_result=result['task_result_proto'],
                     )
                     pe_logs.append(playbook_execution_log)
 
         except Exception as exc:
             logger.error(f"Error occurred while running playbook: {exc}")
 
-        execution_output: [InterpretationProto] = playbook_execution_result_interpret(InterpreterType.BASIC_I, p_proto,
-                                                                                      pe_logs)
+        execution_output: [InterpretationProto] = playbook_execution_result_interpret(InterpreterType.BASIC_I, p_proto, pe_logs)
+        action_executor(account, workflow.actions[0], execution_output)
+
+    if message_type == WorkflowActionSlackNotificationConfig.MessageType.THREAD_REPLY:
+
+        # Sample alert message
+        channel_id = workflow.entry_points[0].alert_config.slack_channel_alert_config.slack_channel_id.value
+        bot_auth_token = ConnectorKey.objects.get(key_type=ConnectorKeyProto.KeyType.SLACK_BOT_AUTH_TOKEN, is_active=True).key
+        message_ts = SlackApiProcessor(bot_auth_token).send_bot_message(channel_id, 'Hello, this is a test alert message from to show how the enrichment works in reply to an alert.')
```

Comment:
Hello, this is a test alert message from (insert - MyDroid or Doctor Droid here) to show how the enrichment works in reply to an alert.

Reply:
Can't find a way to self tag. Fixed the grammar